### DISCUSSION TITLE
Integration picker alphabet/category shortcuts

### DISCUSSION THREAD

**Discussion by vsnine** [home-assistant/frontend#7543] [Configuration]
Hello,

The integration picker has quite a few entries in it now, it can take a long time to scroll. I think it may be beneficial to perhaps include a option to either filter the list by the starting letter or perhaps by the integration type (or both).

Mock-up: https://imgur.com/a/prJ4udS

Thanks.

**Comment by matthiasdebaat** [comment]
Yeah great suggestion. I'll put it on my list to make a Figma mockup.
